navi/src/main/java/com/trello/navi/Event.java

```diff
@@ -172,6 +173,11 @@
   public static final Event<Void> DESTROY_VIEW = new Event<>(Type.DESTROY_VIEW, Void.class);
 
   /**
+   * Emits {@link Fragment#onViewCreated(View, Bundle)} ()}. Emitted before super().
+   */
+  public static final Event<Bundle> VIEW_CREATED = new Event<>(Type.VIEW_CREATED, Bundle.class);
```

Comment:
Ah, I missed it the first time, but this should actually be a type that emits both the `View` and the `Bundle`, to match the other listeners with multiple parameters.